### PR TITLE
[15.0][ADD] website_sale_wishlist_archive_cron

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
This module adds a cron task that will "Archive" all the wishlist items that the product is inactive.

@qrtl

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Shouldn't this be done as soon as the product is archived to avoid problems of someone ordering the product while archived?

**Comment by AungKoKoLin1997** [comment]
> Shouldn't this be done as soon as the product is archived to avoid problems of someone ordering the product while archived?

I think this may not happen. Even users add to cart the wishlist items that the product is inactive, the cart will be empty. So, they can't order.

**Review by kanda999 — APPROVED** [review]
Function review: It worked as expected

File: website_sale_wishlist_archive_cron/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This module does the following:
+
+* A cron task that will "Archive" all the wishlist items that the product is inactive
```
Comment (yostashiro) [inline website_sale_wishlist_archive_cron/readme/DESCRIPTION.rst]: To reflect https://github.com/OCA/e-commerce/pull/801#issuecomment-1552670897...

```suggestion
This module introduces a scheduled action record that archives wishlist items associated with
inactive (archived) products.

Background:

The default behavior in vanilla Odoo for archived products is as follows:

* Existing wishlist items for the product remain unaffected (they persist in the wishlist).
* When a customer attempts to add the item to their cart, it vanishes from the wishlist and the cart
  remains unchanged.

This behavior can be confusing and may result in customer dissatisfaction.
```

**Comment by yostashiro** [comment]
@AungKoKoLin1997 Thanks! Can you please squash commits.

**Review by yostashiro — APPROVED** [review]
Code review.

**Review by chienandalu — APPROVED** [review]

**Comment by AungKoKoLin1997** [comment]
@pedrobaeza  merge was failed. Can you please try again?

**Pull request merged**